data/org.mate.pluma.gschema.xml.in

```diff
@@ -216,5 +216,25 @@
       <summary>Active plugins</summary>
       <description>List of active plugins. It contains the "Location" of the active plugins. See the .pluma-plugin file for obtaining the "Location" of a given plugin.</description>
     </key>
+    <key name="enable-space-drawer-space" type="b">
+      <default>false</default>
+      <summary>Draw spaces</summary>
+      <description>Whether pluma should draw spaces in the editor window.</description>
+    </key>
+    <key name="enable-space-drawer-tab" type="b">
+      <default>false</default>
+      <summary>Draw tabs</summary>
+      <description>Whether pluma should draw tabs in the editor window.</description>
+    </key>
+    <key name="enable-space-drawer-newline" type="b">
+      <default>false</default>
+      <summary>Draw newline</summary>
+      <description>Whether pluma should draw newlines in the editor window.</description>
+    </key>
+    <key name="enable-space-drawer-nbsp" type="b">
+      <default>false</default>
+      <summary>Draw nbsp</summary>
+      <description>Whether pluma should draw 'Not Breaking SPace' in the editor window.</description>
```

Comment:
```
<summary>Draw non-breaking spaces</summary>
<description>Whether pluma should draw non-breaking space characters in the editor window.</description>
```

Reply:
Yes, it makes sense

Reply:
> Also, I'm not entirely sure how to test the nbsp toggle. There's no checkbox in the UI for it, and now sure how to type one...

I still have to understand which are the nbsp spaces...
Currently it is possible to change the setting using gsetting; if you are able to point me an use case I will implement the UI for it.

Reply:
I honestly have no idea about `nbsp`, I've used it for web development by adding a `&nbsp;`, but that only becomes a non-breaking space upon rendering, so not sure how to represent this in a text editor... I would suggest ignoring nbsp's for now and focus on the spaces/tabs/newlines so that we can get this merged in.

Reply:
I already implemented it. This would be settable via dconf